.github/workflows/nightly.yaml

```diff
@@ -4,31 +4,10 @@ on:
   workflow_dispatch:
   schedule:
     - cron: '0 0 * * *'
-
-env:
-  REGISTRY_URL: localhost:5000
+  pull_request:
+    branches:
+      - main
 
 jobs:
-  slow-tests:
-    name: "slow tests"
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/cache@v3
-        id: restore-build
-        with:
-          path: stacker
-          key: ${{ github.sha }}
-      - name: zot registry
-        env:
-          ZOT_HOST: localhost
-          ZOT_PORT: 5000
-        run: |
-          # start a zot instance (minimal)
-          podman run -d -p ${ZOT_PORT}:${ZOT_PORT} ghcr.io/project-zot/zot-minimal-linux-amd64:latest
-          # check if reachable
-          while true; do x=0; curl -f http://${REGISTRY_URL}/v2/ || x=1; if [ $x -eq 0 ]; then break; fi; sleep 1; done
-      - name: test
-        env:
-          SLOW_TEST: true
-        run: |
-          make check
+  build:
+    uses: ./.github/workflows/build.yaml
```

Comment:
where is "slow-test" actually set to true?

Reply:
> where is "slow-test" actually set to true?

It defaults to true. (.github/workflows/build.yaml).

it seems that "run all the tests" is a better default.  That way in a release and in a nightly we get all the tests run. the only place we don't get them all run is in the 'ci.yaml' caller.